orchestration/OrchestrationService/ModuleConfigurationDeployment.ps1

```diff
@@ -30,14 +30,15 @@ $deploymentService = $null;
 $cacheDataService = $null;
 $auditDataService = $null;
 $moduleStateDataService = $null;
-$archetypeInstanceBuilder = $null;
+$configurationBuilder = $null;
 $factory = $null;
 $defaultLocation = "West US";
 $notSupportedVersion = 1.9;
 $defaultSupportedVersion = 2.0;
 $defaultModuleConfigurationsFolderName = "modules";
 $defaultTemplateFileName = "deploy.json";
 $defaultParametersFileName = "parameters.json";
+$toolkitConfigurationFileName = "toolkit.config.json";
```

Comment:
Per our conversation, this should be part of the bootstrap. The bootstrap process should know where to find a config file and to use defaults if no file is found (local storage and local audit should be used as default)

Reply:
Moved this to Invoke-Bootstrap function.